javascript/javascript2/week1/medianAndAverage.js

```diff
@@ -0,0 +1,53 @@
+function getMedian(values)//here median function
```

Comment:
This didn't work, i got an error when i ran this `ReferenceError: averageVal is not defined`